.github/workflows/main.yaml

```diff
@@ -134,6 +156,15 @@ jobs:
     - name: Test Python
       run: bash ./pythia test
 
+    - name: Upload testing package
+      env:
+        BIN_KEY: ${{ secrets.SFTPPLUS_BIN_KEY }}
+      run: |
+        touch bin.key
+        chmod 600 bin.key
+        echo "$BIN_KEY" > bin.key
+        sftp -b sftp_batch -i bin.key -o StrictHostKeyChecking=no bin@bin.chevah.com
```

Comment:
maybe we should also have the `bin.chevah.com` SFTP server fingerpint in the GitHub secrets so that we don't accept any server.

`StrictHostKeyChecking=no` would be ok if we would not  run this on a VM for which the cache is erased at each build. 

This is similar to SFTPlus `ssh_server_identity=set-on-first-connection` is just that with GitHub action VM each run is the first connection.

Reply:
btw. for adding SSH key there is this action  https://github.com/marketplace/actions/install-ssh-key

I am not saying that we should use it...just check if there are any security issues that we miss and might be handled by that action.

I don't see the SSH Server fingerprint issue solved there.

Reply:
I also see this action https://github.com/marketplace/actions/setup-ssh but I don't see any option to add the SSH server public key.

Instead of SSH server fingerprint, we should keep in GitHub Secrets the public key.... from it, you can generate a fingerprint.

Reply:
I see this action which has fingerprint option ... but default is to skip it ... WHY? :)

https://github.com/appleboy/ssh-action

Reply:
and a possible solution for this - https://stackoverflow.com/questions/60477061/github-actions-how-to-deploy-to-remote-server-using-ssh ... with RAW `known_hosts` content in secrets :)

Reply:
Thanks, great suggestion! In hindsight, it's obvious, like any great idea… :-]